composer.json

```diff
@@ -33,6 +33,13 @@
     }
   },
   "require": {
+    "ext-bcmath": "*",
+    "ext-bz2": "^1.0.6",
+    "ext-gd": "^1.2.50",
+    "ext-mysqli": "^5.0.12",
+    "ext-opcache": "^7.0.28",
+    "ext-simplexml": "*",
+    "ext-tidy": "^7.0.28",
```

Comment:
I think we can just provide `*` as version specifier. We just want these extensions to be installed and `composer` to perform the dependencies check.

Reply:
Fixed. However it doesn't work in that way. Composer isn't checking whether extension is really installed :(